test/definitions/checks/foreman/check_corrupted_roles_test.rb

```diff
@@ -14,10 +14,16 @@
   end
 
   it 'fails when some roles with corrupted filters detected' do
-    assume_feature_present(:foreman_database, :query => [{ 'role_id' => 5 }])
+    assume_feature_present(:foreman_database, :present? => true)
```

Comment:
I don't think that it is required to expect `:present?` for `feature(:foreman_database)`. Or am i missing something here?

Reply:
Yes, no need of `:present?` for `feature(:foreman_database)`. Removing `assume_feature_present`.